common/ManagementTestShared/Redesign/ManagementRecordedTestBase.cs

```diff
@@ -34,14 +34,14 @@ public abstract class ManagementRecordedTestBase<TEnvironment> : RecordedTestBas
         private ArmClient _cleanupClient;
         private bool _waitForCleanup;
 
-        protected ManagementRecordedTestBase(bool isAsync, bool useLegacyTransport = false) : base(isAsync, useLegacyTransport: useLegacyTransport)
+        protected ManagementRecordedTestBase(bool isAsync, bool useLegacyTransport = true) : base(isAsync, useLegacyTransport: useLegacyTransport)
         {
             SessionEnvironment = new TEnvironment();
             SessionEnvironment.Mode = Mode;
             Initialize();
         }
 
-        protected ManagementRecordedTestBase(bool isAsync, RecordedTestMode mode, bool useLegacyTransport = false) : base(isAsync, mode, useLegacyTransport)
+        protected ManagementRecordedTestBase(bool isAsync, RecordedTestMode mode, bool useLegacyTransport = true) : base(isAsync, mode, useLegacyTransport)
```

Comment:
Do we need to switch everything off? I believe the cert issue and .NET 6 SDK issues have been addressed. The test failures you are running into in this PR would not be fixed by turning off the Test Proxy.

Reply:
Making the switch global so that turning it on / off is in one place and we don't have to worry about missing one RP when we need to toggle.

Reply:
Not sure I follow - the switch is already global, but with this change it is being turned off everywhere.